package.json

```diff
@@ -19,11 +19,21 @@
     "web-vitals": "^1.0.1",
     "yup": "^0.32.11"
   },
+  "lint-staged": {
+    "src/**/*.ts*": [
+      "yarn prettier:fix",
+      "yarn lint"
+    ]
+  },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint src",
+    "prettier:fix": "prettier --write",
+    "prettier:check": "prettier --check",
+    "prepare": "husky install"
```

Comment:
Do you need to run `yarn prepare` to use husky?

Reply:
Not sure, but let's leave it for now.